rules/dispatch_test.go

```diff
@@ -685,7 +685,7 @@ var _ = Describe("Dispatch chains", func() {
 				func(names []string, expectedChains []*iptables.Chain) {
 					input := convertToInput(names, expectedChains)
 					// Note: order of chains and rules should be deterministic.
-					Expect(renderer.HostDispatchChains(input, true)).To(Equal(expectedChains))
+					Expect(renderer.HostDispatchChains(input, "", true)).To(Equal(expectedChains))
```

Comment:
Should have next tests here to verify the non `""` case

Reply:
Presume you mean **new** tests?